xml/System/String.xml

```diff
@@ -7984,12 +7991,13 @@ The `comparisonType` parameter is a <xref:System.StringComparison> enumeration m
   
  You can use the <xref:System.String.IsNullOrWhiteSpace%2A> method to test whether a string is `null`, its value is <xref:System.String.Empty?displayProperty=nameWithType>,  or it consists only of white-space characters.  
   
-## What is a null string?
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```

Comment:
all the samples for IsNullOrEmpty should be converted to interactive. Some need to move the snippet tag though to work. but then I'd move the note to the top of the examples section instead.

Reply:
Done in https://github.com/dotnet/samples/pull/1171